packages/viewport/src/index.js

```diff
@@ -189,3 +189,32 @@ export function subscribeIsDesktop( listener ) {
 export function getWindowInnerWidth() {
 	return isServer ? SERVER_WIDTH : window.innerWidth;
 }
+
+/******************************************/
+/*       Vertical Scroll Experiment       */
+/******************************************/
+
+//TODO: To be refactored using above using the DESKTOP_BREAKPOINT constant
+export function isTabletResolution() {
+	if ( ! isServer ) {
+		return window.innerWidth < 1040;
+	}
+	return false;
+}
+
+export const DEVICE_MOBILE = 'MOBILE';
```

Comment:
Nitpick: It would be consistent with other analytics props if this value is in lowercase instead of all uppercase.

Reply:
Ah yes I will change this! 👍